modules/tinymce/src/core/main/ts/init/InitContentBody.ts

```diff
@@ -450,7 +463,12 @@ const contentBodyLoaded = (editor: Editor): void => {
   preInit(editor);
 
   setupRtcThunk.fold(() => {
-    loadContentCss(editor).then(() => initEditorWithInitialContent(editor));
+    const cancelProgress = startProgress(editor);
+    loadContentCss(editor).then(() => {
+      initEditorWithInitialContent(editor);
+      cancelProgress();
+      editor.setProgressState(false);
```

Comment:
Put this into the `cancelProgress()` function please

Reply:
moved